### PR TITLE
keep tab order when opening new tab

### PR DESCRIPTION
**Pull request by apfelchips**
see: https://gitlab.com/mjanetmars/switch-container/-/blob/master/switch_container.js

Sorry I couldn't test it, FF says the built .zip --> .xpi  is corrupted. 

xpinstall.signatures.required -> false
I'm on macOS Catalina & FF dev 75.0b11.
any Idea why?

### CODE REVIEW

**Comment by apfelchips** [comment]
ok I got it to install and it works perfectly, please merge this PR.

I've had to go to `about:debugging#/runtime/this-firefox` -> Load Temporary Add-on

File: src/opener.js [hunk 1]
```diff
@@ -102,14 +102,29 @@ async function createContainer(params) {
...
+		cookieStoreId: container.cookieStoreId,
+		url: params.url,
+		index: currentTab.index+1,
+		pinned: params.pinned,
+		openInReaderMode: params.openInReaderMode,
+	})
+	await browser.tabs.remove(currentTab.id)
+}
+
+async function currentTab(container, params) {
```
Comment (honsiorovskyi) [inline src/opener.js:115]: This function is not used anywhere, can it be removed?

Comment (apfelchips) [reply src/opener.js]: yeah I thought I'd have to rewrite the original function, and forgot the copy. Fixed :)

File: CONTRIBUTORS [hunk 2]
```diff
@@ -4,4 +4,4 @@ Extension code by:
 Extension icons by:
 - Pixel perfect (https://www.flaticon.com/authors/pixel-perfect from https://www.flaticon.com/)
 - Freepik (https://www.flaticon.com/authors/freepik) from https://www.flaticon.com/
-- Denys Honsiorovskyi (https://github.com/honsiorovskyi)
\ No newline at end of file
+- Denys Honsiorovskyi (https://github.com/honsiorovskyi)
```
[the remark this answers is not in the document]
Comment (apfelchips) [reply CONTRIBUTORS:7]: Thanks.

**Comment by apfelchips** [comment]
This PR should be ready now.

**Comment by honsiorovskyi** [comment]
Hey @apfelchips!

Thanks a lot for your contribution and sorry for taking too much time to approve your PR!

Regarding the PR itself, I was working on a brand new version lately, and I've manually incorporated your changes there (and also added you to the contributors list). There now there's no point in this PR as is, since it's already in master. Please feel free to review it if you want.

I will close the PR, but I want to thank you again for spending your time working on this, and I hope you enjoy the new version :)

**Pull request closed without merging**